lib/prm/web/controllers/doctor_controller.ex

```diff
@@ -0,0 +1,43 @@
+defmodule PRM.Web.DoctorController do
+  @moduledoc false
+
+  use PRM.Web, :controller
+
+  alias PRM.DoctorAPI
+  alias PRM.Doctor
+
+  action_fallback PRM.Web.FallbackController
+
+  def index(conn, _params) do
+    doctors = DoctorAPI.list_doctors()
+    render(conn, "index.json", doctors: doctors)
+  end
+
+  def create(conn, %{"doctor" => doctor_params}) do
+    with {:ok, %Doctor{} = doctor} <- DoctorAPI.create_doctor(doctor_params) do
+      conn
+      |> put_status(:created)
+      |> put_resp_header("location", doctor_path(conn, :show, doctor))
+      |> render("show.json", doctor: doctor)
+    end
+  end
+
+  def show(conn, %{"id" => id}) do
+    doctor = DoctorAPI.get_doctor!(id)
+    render(conn, "show.json", doctor: doctor)
+  end
+
+  def update(conn, %{"id" => id, "doctor" => doctor_params}) do
+    doctor = DoctorAPI.get_doctor!(id)
```

Comment:
I'd like to keep it the way it is now, e.g. on par with other actions in this controller.